src/ui/public/chrome/chrome.html

```diff
@@ -1,71 +1,48 @@
 <div class="content" chrome-context >
   <!-- TODO: These config dropdowns shouldn't be hard coded -->
-  <nav class="app-links-wrapper" ng-show="chrome.getVisible()">
```

Comment:
The Styleguide only talks about doing multiple lines when you exceed our character limit. There's nothing about *not* breaking it down earlier, but adding all those lines makes it harder to quickly look at the markup; less lines, more readable markup, in my opinion anyway.